app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/audit/onboarding/BusinessPartnerRecordNameMatchAttemptEvent.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.audit
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.audit.onboarding
```

Comment:
can we have `onboarding.audit` instead of `audit.onboarding`? It'd be neater

Reply:
fixed